source/tmf-services/TMF638_Service_Inventory/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "openapi-petstore",
+  "version": "1.0.0",
+  "description": "This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.",
```

Comment:
```suggestion
  "name": "TMF638-Service-Inventory",
  "version": "1.0.0",
  "description": "",
```

Reply:
These are the original sources from the Sample-Implementation on tmforum.org.
So, maybe this should be fixed there also   ;-)